docs/7.Scan Examples/Sca.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: default
+published: true
+title: SCA scanning
+nav_order: 20
+---
+
+# SCA scanning
+Checkov is an SCA (Software Composition Analysis) tool. This means it scans package files closed source images for Common Vulnerabilities and Exposures (CVEs).
+
+You can [Find here](https://docs.bridgecrew.io/docs/open-source-vulnerability-scanning) the full list of the supported package manager types.
```

Comment:
`You can find the full list of the supported package manager types [here]((https://docs.bridgecrew.io/docs/open-source-vulnerability-scanning).`